Grammar/python.gram

```diff
@@ -390,7 +390,7 @@ atom[expr_ty]:
     | &'{' (dict | set | dictcomp | setcomp)
     | '...' { _Py_Constant(Py_Ellipsis, NULL, EXTRA) }
 
-strings[asdl_seq*] (memo): a=STRING+ { concatenate_strings(p, a) }
+strings[expr_ty] (memo): a=STRING+ { concatenate_strings(p, a) }
```

Comment:
I am still sad that `gcc` does not complain about this....

Reply:
It actually did on my PC, that's how I noticed it. It was only a warning, but I guess it'd be an error with `-Wincompatible-pointer-types`.